ambry-network/src/main/java/com.github.ambry.network/Selector.java

```diff
@@ -316,6 +320,13 @@ public void poll(long timeoutMs, List<NetworkSend> sends)
         try {
           if (key.isConnectable()) {
             handleConnect(key, transmission);
```

Comment:
Now this method only calls `transmission.finishConnect()`, maybe we could avoid this method. Also `key` argument is not being used.